apps/calendar/test/marionette/lib/views/week.js

```diff
@@ -69,10 +69,13 @@ Week.prototype = {
     });
   },
 
-  waitForHourScrollEnd: function() {
+  waitForHourScrollEnd: function(hour) {
     // if displaying current day it scrolls to current time, if not it scrolls
-    // to 8AM
-    var hour = this.todayDates.length ? new Date().getHours() - 1 : 8;
+    // to 8AM; it should also scroll to the created event, that's why we allow
+    // overriding the `hour`.
+    if (hour == null) {
+      hour = this.todayDates.length ? new Date().getHours() - 1 : 8;
```

Comment:
Some parens here would be helpful :). Not obvious that `-` binds more strongly than `:`